docs/3.0.0-beta.x/guides/reverse-proxy.md

```diff
@@ -0,0 +1,210 @@
+# Reverse Proxy
+
+When in production, you may want to use Strapi behind a reverse proxy like Nginx. In this guide we will give few **examples** of common use-cases for hosting on an **Ubuntu server**.
+
+## Behind a domain name (mywebsite.com)
+
+- You want to put Strapi on your server with the domain name `mywebsite.com`.
+- You want the API to be accessible at: `https://mywebsite.com`
```

Comment:
I think it would be better to use `api.mywebsite.com` here as users will have issues if they try to use this config with both a frontend and Strapi running on this domain which is the most likely usecase else most will either do sub-folder or sub-domain.